sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorRecordedTestSanitizer.cs

```diff
@@ -1,35 +1,26 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System.Text.RegularExpressions;
 using Azure.Core.TestFramework;
 using Azure.Core.TestFramework.Models;
 
 namespace Azure.AI.MetricsAdvisor.Tests
 {
     public class MetricsAdvisorRecordedTestSanitizer : RecordedTestSanitizer
     {
-        private static Regex emailRegex = new Regex(@"\w+@microsoft.com", RegexOptions.Compiled);
         public MetricsAdvisorRecordedTestSanitizer()
         {
             SanitizedHeaders.Add(Constants.SubscriptionAuthorizationHeader);
             SanitizedHeaders.Add(Constants.ApiAuthorizationHeader);
-            AddJsonPathSanitizer("$..password");
-            AddJsonPathSanitizer("$..certificatePassword");
-            AddJsonPathSanitizer("$..clientSecret");
-            AddJsonPathSanitizer("$..keyVaultClientSecret");
-            AddJsonPathSanitizer("$..apiKey");
-            AddJsonPathSanitizer("$..accountKey");
-            AddJsonPathSanitizer("$..authHeader");
-            AddJsonPathSanitizer("$..httpHeader");
+            JsonPathSanitizers.Add("$..password");
+            JsonPathSanitizers.Add("$..certificatePassword");
+            JsonPathSanitizers.Add("$..clientSecret");
+            JsonPathSanitizers.Add("$..keyVaultClientSecret");
+            JsonPathSanitizers.Add("$..apiKey");
+            JsonPathSanitizers.Add("$..accountKey");
+            JsonPathSanitizers.Add("$..authHeader");
+            JsonPathSanitizers.Add("$..httpHeader");
             BodyRegexSanitizers.Add(new BodyRegexSanitizer(@"\w+@microsoft.com", "foo@contoso.com"));
         }
-
-        public override string SanitizeTextBody(string contentType, string body)
-        {
-            body = emailRegex.Replace(body, "foo@contoso.com");
```

Comment:
Are these sanitized by default now?

Reply:
This was already updated to use the new sanitization approach, but I just forgot to remove this.